src/taipy/gui/_default_config.py

```diff
@@ -62,11 +62,11 @@
         "font_family": "Lato, Arial, sans-serif",
 
         # DEFINING SHAPES
-        # Base border radius
-        "border_radius": "8px",
+        # Base border radius in px
+        "border_radius": 8,
```

Comment:
Thanks for this addition. TS has been screaming at me on the studio side on this value